File: HuaweiVcLib/src/main/java/com/hw/huaweivclib/activity/CallOutActivity.java
package com.hw.huaweivclib.activity;

import android.os.Bundle;
import android.text.TextUtils;
import android.view.View;
import android.widget.TextView;

import com.huawei.opensdk.callmgr.CallMgr;
import com.hw.huaweivclib.R;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

/**
 * 去电界面
 */
public class CallOutActivity extends BaseMediaActivity {
    @Override
    public void onBackPressed() {
//        super.onBackPressed();
    }

    @Override
    public int bindLayout() {
        return R.layout.activity_call_out;
    }

    @Override
    public void initView(@Nullable Bundle savedInstanceState, @NotNull View contentView) {

    }

    @Override
    public void doBusiness() {

    }

    @Override
    public void setListeners() {
        ((TextView) findViewById(R.id.tv_number)).setText(TextUtils.isEmpty(String.valueOf(mCallNumber)) ? "" : String.valueOf(mCallNumber));

        findViewById(R.id.tv_hang_up).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                CallMgr.getInstance().endCall(mCallID);
                finish();
            }
        });

        ((TextView) findViewById(R.id.tv_number)).setText(TextUtils.isEmpty(mCallNumber) ? "" : mCallNumber);

    }
}

File: vclibrary/src/main/java/com/zxwl/vclibrary/activity/CallOutActivity.kt
package com.zxwl.vclibrary.activity

import android.text.TextUtils
import android.view.View
import android.widget.TextView
import com.huawei.opensdk.callmgr.CallMgr
import com.zxwl.vclibrary.R
import kotlinx.android.synthetic.main.activity_call_out.*
import me.jessyan.autosize.internal.CancelAdapt

/**
 * 外呼界面
 */
class CallOutActivity : BaseMediaActivity(), CancelAdapt {

    override fun getLayoutId(): Int = R.layout.activity_call_out

    override fun setListener() {
        super.setListener()

        tvNumber.text = mCallNumber.toString() ?: ""

        if ("+991117".equals(mCallNumber)) {
            tvCallContent.text = "正在加入会议..."
        } else {
            tvCallContent.text = if (TextUtils.isEmpty(mCallNumber)) "" else mCallNumber
        }

        tvHangUp.setOnClickListener {
            CallMgr.getInstance().endCall(mCallID)
            finish()
        }
    }
}